scripts/rain_data.py

```diff
@@ -0,0 +1,19 @@
+import argparse
```

Comment:
yeah. I can remove the rain files and submit another pull request as we don't need them